### ISSUE TITLE
Grunt 0.4 Release

### ISSUE THREAD

**Issue by tkellen** [pajtai/grunt-php-set-constant#7]
I'm posting this issue to let you know that we will be publishing Grunt 0.4 on Monday, February 18th.

If your plugin is not already Grunt 0.4 compatible, would you please consider updating it? For an overview of what's changed, please see our [migration guide](https://github.com/gruntjs/grunt/wiki/Upgrading-from-0.3-to-0.4).

If you'd like to develop against the final version of Grunt before Monday, please specify `"grunt": "0.4.0rc8"` as a devDependency in your project. After Monday's release, you'll be able to use `"grunt": "~0.4.0"` to actually publish your plugin. If you depend on any plugins from the grunt-contrib series, please see our [list of release candidates](https://github.com/gruntjs/grunt/wiki/Plugin-Release-Candidate-Versions) for compatible versions. All of these will be updated to final status when Grunt 0.4 is published.

Also, in an effort to reduce duplication of effort and fragmentation in the developer community, could you review the [grunt-contrib](http://github.com/gruntjs) series of plugins to see if any of your functionality overlaps significantly with them?  Grunt-contrib is community maintained with 40+ contributors&mdash;we'd love to discuss any additions you'd like to make.

Finally, we're working on a new task format that doesn't depend on Grunt: it's called [node-task](/tkellen/node-task).  Once this is complete, there will be one more conversion, and then we'll never ask you to upgrade your plugins to support our changes again.  Until that happens, thanks for bearing with us!

If you have any questions about how to proceed, please respond here, or join us in [#grunt on irc.freenode.net](irc://irc.freenode.net/#grunt).

Thanks, we really appreciate your work!

**Comment by ghost** [comment]
Any updates on that? I want to upgrade to Grunt 0.4, but want to keep using your plugin.

Don't know if I could be of any help, because I don't really know NodeJS, but let me know if can lend you a hand!

**Comment by pajtai** [comment]
@falzhobel - v0.0.13 should be good to go! I just made a few updates and ran the tests and they passed.

Let me know if you run into any problems.

**Comment by ghost** [comment]
Actually I just updated to Grunt 0.4, and added version 0.0.13 as a dependency for grunt-php-set-constant and I am getting a 404 from the npm registry : 

```
npm http 404 https://registry.npmjs.org/grunt-php-set-constant/-/grunt-php-set-constant-0.0.13.tgz
npm ERR! fetch failed https://registry.npmjs.org/grunt-php-set-constant/-/grunt-php-set-constant-0.0.13.tgz
```

Thanks for letting me know !

**Comment by pajtai** [comment]
@falzhobel Yeah, I can reproduce. It seems that 0.0.13 has some issues on NPM. I just pushed 0.0.14, and I can download that without errors. Try that one.

**Comment by ghost** [comment]
Yup worked like a charm, good job and thanks again !
